tephra-core/src/test/java/co/cask/tephra/TransactionContextTest.java

```diff
@@ -54,7 +53,7 @@
   @BeforeClass
   public static void setup() throws IOException {
     final Configuration conf = new Configuration();
-    conf.set(TxConstants.Persist.CFG_TX_SNAPHOT_CODEC_CLASSES, DefaultSnapshotCodec.class.getName());
+    conf.set(TxConstants.Persist.CFG_TX_SNAPHOT_CODEC_CLASSES, SnapshotCodecV2.class.getName());
```

Comment:
I change it to V2 because that's what the `@deprecate` javadoc in the `DefaultSnapshotCodec` said. Do you think I should change the javadoc in as well?